processes/ingest/publisher_backlist.py

```diff
@@ -1,50 +1,50 @@
 import os
 from services import PublisherBacklistService
 
-from ..core import CoreProcess
 from logger import create_log
-from managers import S3Manager
+from managers import DBManager, S3Manager
+from ..record_buffer import RecordBuffer
+from .. import utils
 
 logger = create_log(__name__)
 
-class PublisherBacklistProcess(CoreProcess):
+class PublisherBacklistProcess():
     def __init__(self, *args):
-        super(PublisherBacklistProcess, self).__init__(*args[:4])
+        self.process = args[1]
```

Comment:
this should be args[0]

Reply:
oops lol

Reply:
Good catch - I've been thinking about creating a utils function to parse the args

Reply:
I think that's a good idea!